bundler/lib/bundler/inline.rb

```diff
@@ -41,6 +41,7 @@ def gemfile(install = false, options = {}, &gemfile)
     Bundler.instance_variable_set(:@bundle_path, Pathname.new(Gem.dir))
     old_gemfile = ENV["BUNDLE_GEMFILE"]
     Bundler::SharedHelpers.set_env "BUNDLE_GEMFILE", "Gemfile"
+    Bundler::SharedHelpers.major_deprecation 2, "'install' argument. The installation will happen for missing gems. It is not necessary pass a 'true' value." if install
```

Comment:
I feel this deprecation is going to affect many usages of `bundler/inline`, let's improve a bit the message. Maybe

> The optional `install` parameter to the `gemfile(install = false, &block)` helper is getting removed because regardless of what you pass in there, it still installs missing gems. Remove the explicitly `install` parameter to get rid of this message.

And since this requires users changing their code, I think we'll want to include the exact path and line that needs to be changed. Please research on how to do that, and let me know if you need help.

Reply:
Hey @deivid-rodriguez just to be clear, when you say 'exact path and line', you mean the path and line of `inline.rb`? Because I was wondering if using `Dir.pwd:<line>` in the deprecated message would work as you want.

Reply:
Basically we need to print the exact file and line where end users are using `gemfile(true) {}` in their code. Actually, I just saw that the major deprecation helper includes a parameter just for this 🎉. You have to use `SharedHelpers.major_deprecation 2, "<message>", print_caller_location: true`.

Reply:
Ok. I'll push the code now.